drivers/Makefile

```diff
@@ -16,7 +16,8 @@ INCLUDES = -I../include/ \
 	 -I./axi_core/axi_dmac \
 	 -I./axi_core/clk_axi_clkgen \
 	 -I./platform/xilinx \
-	 -I../projects/adrv9009/src/devices/adi_hal
+	 -I../projects/adrv9009/src/devices/adi_hal \
+	 -I../libraries/iio
```

Comment:
Is this change related to this commit?

Reply:
If I move the change to different commit, the build for this particular commit will fail.

Reply:
Sholdn't then the iio file be added in /noos/iio ?

Reply:
Based on a previous meeting discussion, we decided to add the IIO driver in the driver's folder.

Reply:
Ok, I missed the meeting :)

Reply:
Anyway, you can add a pr for this change, in #863 will be needed too.